public/js/interface.js

```diff
@@ -0,0 +1,134 @@
+const game = new Game();
+let frame = new Frame();
+let i = 0;
+
+$(document).ready(() => {
+  const next = $('.next-frame');
+  const pins = $('.pins');
+
+  $('.begin button').click(function jQuery() {
+    $(this).parent().addClass('hidden');
+    $('.game').removeClass('hidden');
+    $(next).addClass('hidden');
+  });
+
+  for (i; i <= 10; i++) {
+    if (i === 0) {
+      pins.append($(`<span data-id="${i}" class="pin-button">0</span>`));
+    } else {
+      pins.append($(`<span data-id="${i}" class="pin-button flaticon-bowling-pin" title="${i}"></span>`));
+    }
+  }
+
+  let count = 0;
+  $('.pin-button').click(function jQuery() {
+    count += 1;
+    let i = 0;
+
+    const id = $(this).data('id');
+
+    for (i; i < id; i++) {
+      $('.pins .pin-button:not(.hidden)').last().addClass('hidden');
+    }
+
+    frame.roll(id);
+
+    if (count === 1) {
+      $('.first-roll').last().html(id);
+    } else if (count === 2) {
+      $('.second-roll').last().html(id);
+    } else if (count === 3) {
+      $('.third-roll').last().html(id);
+    }
+
+    if (game.frames.length < 9) {
```

Comment:
Big blocks of nested if/else conditions like this can be difficult to follow and make it hard for other people to understand what your code is intended to do.  Can you think of any ways in which you could break it up a bit?  You'll have a chance to practice this in week 10, as one of the exercises you'll do is all about doing exactly this.